app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1> posts are found here</h1>
+<div class="user-posts-page-container">
+ <%= render "shared/user_details", user: @user %>
+  <div class="post-comment-container">
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id), class:'nav-link' do%>
+       <%= render "shared/post_details", post: post %>
```

Comment:
- [ ] @Ibtisam34 kindly fix the links in lines 2 and 6 they are currently wrong making the app break, kindly also ensure your posts section aligns well with other section, also position of comments and likes counter
![image](https://user-images.githubusercontent.com/61408860/225056178-652eb09b-5042-4f6a-80cf-732994774c4d.png)